lib/findAccessorMinMax.js

```diff
@@ -9,6 +10,17 @@ var readBufferComponentType = require('./readBufferComponentType');
 
 module.exports = findAccessorMinMax;
 
+/**
+ * Finds the min and max for an accessor in gltf.
+ *
+ * @param {Object} gltf A javascript object holding a glTF hierarchy.
+ * @param {Object} accessor The accessor object from gltf to read.
+ * @returns {{min: Array, max: Array}} min holding the array of minimum values and max holding the array of maximum values.
+ *
+ * gltf must be initialized for the pipeline.
+ * @see addPipelineExtras
+ * @see loadGltfUris
+ */
 function findAccessorMinMax(gltf, accessor) {
```

Comment:
Generally, we do not use abbreviations in public APIs, but this is what glTF calls them so it is OK here.